src/bin/repl/mod.rs

```diff
@@ -97,6 +97,56 @@ impl<'s> Repl<'s> {
     }
 
     fn execute_code(&mut self, code: &str) {
-        let program = check_semantics(code, self.opt.clone());
+        let module = initialize_jit_module();
+        let expr = match analyze(code, Parser::expr, PureAnalyzer::expr) {
+            Ok(mut expr) => {
+                expr.ctype = types::Type::Int(true);
+                expr
+            }
+            Err(err) => {
+                println!("error: {}", err.data);
+                return;
+            }
+        };
+        let function_type = types::FunctionType {
+            return_type: Box::new(types::Type::Int(true)),
+            params: vec![],
+            varargs: false,
+        };
+        let qualifiers = hir::Qualifiers {
+            volatile: false,
+            c_const: false,
+            func: hir::FunctionQualifiers {
+                inline: false,
+                no_return: false,
+            },
+        };
+
+        let main = hir::Variable {
+            ctype: types::Type::Function(function_type),
+            storage_class: data::StorageClass::Extern,
+            qualifiers,
+            id: saltwater::InternedStr::get_or_intern("main"),
+        };
+
+        let span = expr.location;
+        let stmt = span.with(hir::StmtType::Return(Some(expr)));
+        let init = hir::Initializer::FunctionBody(vec![stmt]);
+        let decl = hir::Declaration {
+            symbol: main.insert(),
+            init: Some(init),
+        };
+        let (result, _warns) = ir::compile(module, vec![span.with(decl)], false);
+
+        if let Err(err) = result {
+            println!("failed to compile: {}", err.data);
+            return;
+        }
+
+        let module = result.unwrap();
```

Comment:
Try to avoid `unwrap` if possible.
```suggestion
        let module = match result {
            Err(err) => {
                println!("failed to compile: {}", err.data);
                return;
            }
            Ok(module) => module),
        };
```